src/main/java/ch/jalu/configme/properties/types/PropertyType.java

```diff
@@ -0,0 +1,54 @@
+package ch.jalu.configme.properties.types;
+
+import ch.jalu.configme.beanmapper.Mapper;
+import ch.jalu.configme.resource.PropertyReader;
+
+public interface PropertyType<T> {
+
+    T get(PropertyReader reader, String path);
+
+    T convert(Object object);
+
+    Class<T> getType();
```

Comment:
If we dropped the array property type we'd be able to remove this method :P Or the array property type could require a Class object on it directly?

Reply:
Yes, `getType()` using is only in ArrayProperty